dev/com.ibm.ws.security.oidc.client_fat/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.rp/configs/rp_server_withOpStub_encrypt.xml

```diff
@@ -0,0 +1,1097 @@
+	
+	<authFilter id="authFilterES384RS256">
+		<requestUrl
+			id="requestUrlES384RS256"
+			urlPattern="/simple/SignES384EncryptRS256"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignES384EncryptRS256"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterES384RS256"
+		signatureAlgorithm="ES384"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="es384"
+		keyManagmetnKeyAlias="rs256"
+		sslRef="ssl_allSigAlg"
+	>
+	</openidConnectClient>	
+	
+	<authFilter id="authFilterES512RS256">
+		<requestUrl
+			id="requestUrlES512RS256"
+			urlPattern="/simple/SignES512EncryptRS256"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignES512EncryptRS256"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterES512RS256"
+		signatureAlgorithm="ES512"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="es512"
+		keyManagmetnKeyAlias="rs256"
+		sslRef="ssl_allSigAlg"
+	>
+	</openidConnectClient>	
+
+	<authFilter id="authFilterHS256RS384">
+		<requestUrl
+			id="requestUrlHS256RS384"
+			urlPattern="/simple/SignHS256EncryptRS384"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignHS256EncryptRS384"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		sharedKey="secret"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterHS256RS384"
+		keyManagmetnKeyAlias="rs384"
+		sslRef="ssl_allSigAlg"
+	>
+	</openidConnectClient>
+
+	<authFilter id="authFilterHS384RS384">
+		<requestUrl
+			id="requestUrlHS384RS384"
+			urlPattern="/simple/SignHS384EncryptRS384"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignHS384EncryptRS384"
```

Comment:
Another couple that'll need `signatureAlgorithm` added for HS384/512.

Reply:
I didn't spend much time on new HS algorithms - the issue we have for new signature algs focused on ES and RS - wasn't spending much time on HS until I knew if we were going to support them
Will update as appropriate when I get the runtime code